dev-client/src/screens/SitesScreen/components/search/MapSearch.tsx

```diff
@@ -129,26 +107,29 @@ export default function MapSearch({zoomTo, zoomToUser, toggleMapLayer}: Props) {
             }}
             inputContainerStyle={{borderWidth: 0}} // eslint-disable-line react-native/no-inline-styles
             renderTextInput={() => (
-              <Searchbar
-                onChangeText={newText => {
-                  setQuery(newText);
-                  querySuggestions(newText);
-                }}
-                onFocus={() => {
-                  setHideResults(false);
-                  querySuggestions(query);
-                }}
-                onEndEditing={() => {
-                  setHideResults(true);
-                }}
-                value={query}
-                placeholder={t('search.placeholder')}
-                style={{
-                  ...searchBarStyles.search,
-                  ...searchBarStyles.wideSearchOverride,
-                }}
-                inputStyle={searchBarStyles.input}
-              />
+              <>
+                <Searchbar
+                  onChangeText={newText => {
+                    setQuery(newText);
+                    querySuggestions(newText);
+                  }}
+                  onFocus={() => {
+                    setHideResults(false);
+                    querySuggestions(query);
+                  }}
+                  onEndEditing={() => {
+                    setHideResults(true);
+                  }}
+                  value={query}
+                  placeholder={t('search.placeholder')}
+                  style={{
+                    ...searchBarStyles.search,
+                    ...searchBarStyles.wideSearchOverride,
+                  }}
+                  inputStyle={searchBarStyles.input}
+                />
+                {isOffline && !hideResults ? <MapSearchOfflineBox /> : <></>}
```

Comment:
Why does it matter what `hideResults` is / why not always show the search box when offline?

Reply:
The `OfflineBox` is the alert that shows up underneath the search bar, and it should only show up when the search bar has focus. I can rename some things to clarify this relationship.